EntAppProject/src/main/resources/templates/JBCreateJobListing.html

```diff
@@ -2,37 +2,22 @@
 
 <html>
 
-<head>
-
-<title>Profile </title>
-
-<link rel="stylesheet" href="JobBookCSS.css">
-
-</head>
-
-<body>
- 
-<h1><span>Job</span>Book</h1>
-
-<h2>Create A Job Listing</h2>
-
-<form action="/JBHomescreen">
-Description of Job:
-<br>
-<input style=position:left; type="text" id="userBio">
-<br> 
-
-Payment:
-<br>
-<input style=position:left; type="text" id="userSkills">
-<br>
-
-<button type="submit">Submit</button>
-
-</form>
-
-<!--New Page, JBJobboard.html needs to get added, all submitted jobs will appear there.-->
-
-</body>
-
-</html>
\ No newline at end of file
+    <head>

```

Comment:
I redid the HTML page restructuring from before but with the change of using the `page-title` class on span elements to specify that it should use the style included in `JobBookCSS.css`.